### ISSUE TITLE
Rating af systmer

### ISSUE THREAD

**Issue by ballerupgis** [JosefAssad/SeMaWi#38]
Jeg hørte harddisken her til morgen, hvor de snakkede om programmet rescuetime, som registrerer tidforbruget på forskellige programmer/hjemmesider på ens computer. Det kunne evt. bruges som en mere objektiv indikator i rating af systemer. Tidsforbruget skulle selvfølgelig sammenholdes, men andre typer ratings.

Der findes et API til rescuetime, så vi kan hive oplysningerne ud.

Ulempen er klart at det kræver at brugerne har rescuetime.    

Bare en ide

**Comment by JosefAssad** [comment]
Det ser ret interessant ud. For den enkelte bruger er der klare use cases på at spore produktivitet, men jeg tror use casene på aggregeret niveau (det vil sige, hvis i pusher jeres Rescuetime data ind i SeMaWi) så er der nogle ret fascinerende use cases.

Vi er enige i, at brugt tid i en applikation ikke i sig selv er indikativ på hvor "vigtig" den applikation er, men. Det vil være en yderst indsigtsfuld analyse som viser (lille mockup):

![effektiv](https://cloud.githubusercontent.com/assets/1040956/9852028/4aa11976-5afc-11e5-9d13-8fc38e3d1f8c.png)

Eller, hvis vi finder en formel for "vigtighed":

![effekt2](https://cloud.githubusercontent.com/assets/1040956/9852128/d42733a6-5afc-11e5-9476-18c56f604d2b.png)

Ja det er lidt en ulempe at alle skal have Rescuetime. Jeg tror det kan være en "alle eller ingen" sag, da aggregerede statistikker for tidsforbrug på de to eksempler på use cases nævnt her vil være misvisende hvis der er medarbejdere der ikke bruger Rescuetime. Det kan måske give mening i mindre teams med højtspecialiserede systemer.

Under alle omstændigheder er brugt tid et rigtig interessant facet. Ikke mindst ift. at spore tidsforbrug via systemer over på KLE numre!

**Comment by ballerupgis** [comment]
Det er urealistisk at få rescuetime ud hos alle. Derudover vil der nok også opstå problemer med, at folk føler sig overvåget i deres arbejde. Det er ikke holdbart!

Uanset hvad vil vi gerne lave en rating af systemerne. En vigtigheds-rating er en god ide, men vi skal nok have defineret vigtighed lidt nærmere. Bliver interessant når vi kan sammenholde pris og rating.

**Comment by ballerupgis** [comment]
Vi har lavet et tentativt ratingsytem, det virkeligt interessante er når vi skal diskutere kriterier for ratingen

**Comment by JosefAssad** [comment]
Ja, det har jeg set. Det er godt at starte et sted, og det har vi nu. Nogle overvejelser:

På propertysiden kan det være smart at forklare:

1. Hvad er skala'en?
2. At definere rating. Er det et udtryk for hvor "godt" et system er? Hvis ja, hvad gør et system godt?

Angående tildeling af rating: hvis der er uenighed om rating af et system, hvem bestemmer? der er den klassiske mekanisme som er at man slås lidt om det i Systemets diskussionsside indtil man bliver enige, men det kan også være interessant at videreudvikle property'et således at bestemte personer eller roller må hver indberette rating, og at den samlede rating så er kalkuleret.

Skal der anvendes en form for rating distribution? For at forhindre den mulighed at alle SystemRating scores ligger på samme eller nær samme niveau. (det kan implementeres ved fx. at der kun må tildeles et antal point som svarer til antal systemer * N)

Fordi rating giver så interessante porteføljeanalyser vil det første spørgsmål ved formidling af disse analyser være, hvad er rating'en baseret på? Det er derfor jeg er meget interesseret i at bruge dette indledende arbejde til at bygge en mere skudsikker "rating" mekanisme.

Jeg synes at rating/vigtighed er potentielt det mest interessante facet i hele porteføljen så det er rigtig fint vi er kommet i gang med det. :)
